html/Axit/js/tabs/tab-switching.js

```diff
@@ -0,0 +1,40 @@
+import {components} from './tabs-components.js';
+import {
+  highlightButton,
+  removeButtonActiveClass,
+  hideContent,
+  showElement,
+  findContent,
+  changeSectionHeight,
+} from './helper-functions.js';
+
+export function addTabSwitchingEffect() {
+  components.then(
+      ({tabsButtonsList, contentList, activeTab, tabs, buttons}) => {
+        (function initFirstTab() {
+        // eslint-disable-next-line no-param-reassign
+          activeTab.style.opacity = 1;
+        })();
+
+        changeSectionHeight(activeTab, tabs);
+
+        window.addEventListener('resize', () => {
+          changeSectionHeight(activeTab, tabs);
+        });
+
+        tabsButtonsList.addEventListener('click', (event) => {
+          let target = event.target;
+          if (target.tagName === 'BUTTON') {
```

Comment:
usually it loos a bit more readable when in if call return, at place and write other code in same level
```
if (target.tagName !== 'BUTTON') { return }
//rest code here
```